### PR TITLE
Feature/msgpacktoken test

### LINKED ISSUE #76: Add support for SCALAR type index

### PR DESCRIPTION
**Pull request by roman-kozachenko**
Fixes #76

### CODE REVIEW

File: tarantool/tarantool.lua [hunk 1]
```diff
@@ -44,21 +47,21 @@ box.once('space_TreeIndexMethods', space_TreeIndexMethods)
...
 function log_auth ()
-    local m = 'Authentication attempt'
-    log.info(m)
+	local m = 'Authentication attempt'
+	log.info(m)
 end
 function log_auth_ok (user_name)
-    local m = 'Authenticated user ' .. user_name
-    log.info(m)
+	local m = 'Authenticated user ' .. user_name
```
Comment (aensidhe) [inline tarantool/tarantool.lua]: Do we really want tabs?

File: tests/progaudi.tarantool.tests/TestBase.cs [hunk 2]
```diff
@@ -1,15 +1,16 @@
 using System;
 using System.Threading.Tasks;
+using ProGaudi.Tarantool.Client.Model;
 
 namespace ProGaudi.Tarantool.Client.Tests
 {
     public class TestBase
     {
-        public async Task ClearDataAsync()
+        public async Task ClearDataAsync(string[] spaceNames)
```
Comment (aensidhe) [inline tests/progaudi.tarantool.tests/TestBase.cs]: `params string[]` and refactor accordingly.

File: tests/progaudi.tarantool.tests/progaudi.tarantool.tests.csproj [hunk 3]
```diff
@@ -27,5 +27,9 @@
     <PackageReference Include="Shouldly" Version="2.8.2" />
     <ProjectReference Include="..\..\src\progaudi.tarantool\progaudi.tarantool.csproj" />
   </ItemGroup>
+  
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```
Comment (aensidhe) [inline tests/progaudi.tarantool.tests/progaudi.tarantool.tests.csproj]: That service

File: tarantool/tarantool.lua (hunk 1 above)
Comment (roman-kozachenko) [inline tarantool/tarantool.lua]: As I remember, we decided to use tabs in lua files.

File: tests/progaudi.tarantool.tests/TestBase.cs (hunk 2 above)
Comment (roman-kozachenko) [inline tests/progaudi.tarantool.tests/TestBase.cs]: Done.

File: tests/progaudi.tarantool.tests/progaudi.tarantool.tests.csproj (hunk 3 above)
Comment (roman-kozachenko) [inline tests/progaudi.tarantool.tests/progaudi.tarantool.tests.csproj]: Removed

**Comment by roman-kozachenko** [comment]
Fix #76 and #70

**Pull request merged**